CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
 # Changelog
+## 1.73.0
+* Enhancement - Task Resource Accounting - Remove task serialization and use host resource manager for starting tasks [#3684](https://github.com/aws/amazon-ecs-agent/pull/3684) [#3706](https://github.com/aws/amazon-ecs-agent/pull/3706) [#3700](https://github.com/aws/amazon-ecs-agent/pull/3700) [#3723](https://github.com/aws/amazon-ecs-agent/pull/3723) [#3741](https://github.com/aws/amazon-ecs-agent/pull/3741) [#3747](https://github.com/aws/amazon-ecs-agent/pull/3747) [#3750](https://github.com/aws/amazon-ecs-agent/pull/3750)
```

Comment:
Should this be a feature?

Reply:
Also, `Remove task serialization and use host resource manager for starting tasks` does not sound to me covers the full feature, sorry if I am misunderstanding.

Reply:
Let me update it